File: app/src/main/java/dev/vendicated/vencord/Logger.java
package dev.vendicated.vencord;

import android.util.Log;

public final class Logger {
    private static final String TAG = "Vencord";

    public static void e(String message) {
        Log.e(TAG, message);
    }
    public static void e(String message, Throwable e) {
        Log.e(TAG, message, e);
    }

    public static void w(String message) {
        Log.w(TAG, message);
    }

    public static void i(String message) {
        Log.i(TAG, message);
    }

    public static void d(String message) {
        Log.d(TAG, message);
    }
}

File: app/src/main/java/com/nin0dev/vendroid/Logger.kt
package com.nin0dev.vendroid

import android.util.Log

object Logger {
    private const val TAG = "Vencord"
    @JvmStatic
    fun e(message: String?) {
        Log.e(TAG, message!!)
    }

    @JvmStatic
    fun e(message: String?, e: Throwable?) {
        Log.e(TAG, message, e)
    }

    @JvmStatic
    fun w(message: String?) {
        Log.w(TAG, message!!)
    }

    @JvmStatic
    fun i(message: String?) {
        Log.i(TAG, message!!)
    }

    @JvmStatic
    fun d(message: String?) {
        Log.d(TAG, message!!)
    }
}
